src/libraries/oob-netcurrent-ref.proj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.Build.Traversal">
+  <!-- This project only exists for the /p:RefOnly=true mode.-->
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <!-- Filter ProjectReferences to build the best matching target framework only. -->
+    <FilterTraversalProjectReferences>true</FilterTraversalProjectReferences>
+  </PropertyGroup>
+
+  <!-- Reference all NetCoreAppCurrent out-of-band reference assemblies. -->
```

Comment:
Quick overview:
- sfx-ref.proj => Builds the NetCoreAppCurrent tfm of all shared framework reference projects
- sfx-src.proj => Builds the NetCoreAppCurrent tfm of all shared framework source projects.
- oob-ref.proj => Builds the NetCoreAppCurrent tfm of all out-of-band reference projects.
- oob-src.proj => Builds the NetCoreAppCurrent tfm of all out-of-band source projects.
- oob-all.proj => Builds either all tfms of all all out-of-band projects when the `-allconfigurations` switch is passed in or just the .NETFramework tfms of all the compatible out-of-band projects when the `-f net48` switch is passed in.

> Why do we need these .proj separation? Why can't we do it on the same .proj file?

That's not possible because we have an ordering requirement when composing the shared framework. The shared framework reference assemblies must be available before the shared framework source projects can be compiled. You can't express such a requirement in a single Traversal project.